draining_validation/README.md

```diff
@@ -0,0 +1,10 @@
+# Draining validation
+
+If example-cnf is launched in continuous mode (or emulating continuous mode with a long TRex job duration), we can use this hook to emulate a node draining process that impacts in the worker node where testpmd pod is deployed, then it needs to be reallocated, and consequently packet loss starts to appear.
+
+For this to work, some requirements need to be met:
+
+- example-cnf needs to be deployed in continuous mode or emulating continuous mode, as said before.
+- Direct mode needs to be used, this is not currently validated against loadbalancing mode.
```

Comment:
Just curious, you might know if at some point we will test with load-balancing mode? for me it would be nice to test with LB mode and multiple replicas of testpmd-app, so we can confirm there is no packet loss when one of the replicas is drained : )

Reply:
Ideally, we should test it in loadbalancing mode in the future, and theoretically, packet loss should be 0 since we have multiple testpmd-app pods as you said, but since, currently, the loadbalancer 1) is not behaving properly, and 2) we need to update it to DPDK 23.11, I'm just following what Franck suggested: at least having example-cnf with direct mode, test it, then we check next steps. But yes, I agree that's something we have to explore in the short term!